.github/workflows/analytics_ci.yml

```diff
@@ -1,11 +1,31 @@
-name: Check Analytics Migrations
+name: Analytics Django App CI
 
 on:
   pull_request:
     paths:
       - "analytics/**"
 
 jobs:
+  formatting:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: analytics/
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+
+      - name: Install requirements
+        run: pip install .[dev]
+
+      - name: Format code
+        run: tox -e format
```

Comment:
Wouldn't we want to run `tox -e lint` instead of `format`? Since `format` won't actually contribute the formatted code back to the repo.

Reply:
`lint` runs `flake8` on the code, while `format` runs `black` and `isort`. The codebase isn't compliant with `flake8` rules yet, so I figured just running `black` and `isort` (which is a subset of the formatting rules enforced by `flake8`) is at least a good start. 

`format` will still fail with a non-zero exit code if the code needs to be reformatted.

Reply:
Should we switch `lint` to simply run `black` and `isort`? If the codebase is already compliant with those two anyway, it would largely result in no change.

Reply:
I updated `lint` to run `black` and `isort`